### PR TITLE
doc: update Aurelia integration page

### PR DESCRIPTION
**Pull request by ben-girardet**
# Pull Request

## 📖 Description

Update Aurelia Integration document to reflect changes from Aurelia 2. 

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by EisenbergEffect — APPROVED** [review]

File: packages/web-components/fast-foundation/docs/integrations/aurelia.md [hunk 1]
```diff
@@ -101,89 +88,125 @@ fast-card > fast-button {
...
+  .register(FASTAdapter.customize({withPrefix: 'my-custom-prefix'}) // customized with prefix
+  .start();
+```
+
+Also, in case you have local components that also require two-way binding, you can adjust the adapter before to register it as such:
+
+```ts
+// src/main.ts
+
+import { FASTAdapter } from './aurelia-fast-adapter';
```
Comment (nicholasrice) [inline packages/web-components/fast-foundation/docs/integrations/aurelia.md:133]:
```suggestion
import { FASTAdapter } from 'aurelia-fast-adapter';
```

`aurelia-fast-adapter` is a package right? not a file?

Comment (ben-girardet) [reply packages/web-components/fast-foundation/docs/integrations/aurelia.md]: good catch thanks. This is now fixed

**Comment by EisenbergEffect** [comment]
@nicholasrice Can you give this a quick second review?

**Review by nicholasrice — APPROVED** [review]

**Comment by nicholasrice** [comment]
Thanks @ben-girardet!

**Comment by nicholasrice** [comment]
@ben-girardet can you run `yarn change` from the package root and fill out the prompt? Looks like CI is failing due to no yarn change file.

**Comment by ben-girardet** [comment]
@nicholasrice done, I've ran `yarn change`. I though that it wasn't required for doc changes, but apparently yes...

**Comment by EisenbergEffect** [comment]
Merged! Thanks @ben-girardet !

**Pull request merged**